src/utils.ts

```diff
@@ -181,7 +181,9 @@ export function limitAdhocProviders(
         opts.queries = [];
       }
 
-      let values = (await datasourceHelper.getTagKeys(opts)).slice(0, MAX_ADHOC_VARIABLE_OPTIONS);
+      let values = (await datasourceHelper.getTagKeys(opts))
```

Comment:
We just want to remove it from query interpolation, not remove it as a value to select in the adhocFilters variable :)